include/rocksdb/env.h

```diff
@@ -1563,4 +1565,10 @@ Status NewHdfsEnv(Env** hdfs_env, const std::string& fsname);
 // This is a factory method for TimedEnv defined in utilities/env_timed.cc.
 Env* NewTimedEnv(Env* base_env);
 
+// Returns an instance of logger that can be used for storing informational
+// messages.
+// This is a factory method for EnvLogger declared in logging/env_logging.h
```

Comment:
I know we did it in the function above but it's not a good idea to mention files outside `include/rocksdb` here, because they may not be available to users when it is released.

Reply:
Should i update the comment to say "This is a factory method to create a generic logger that uses env specific methods for logging operations"?

Reply:
I will push the code. We can make the changes in the comment later in a separate PR.